CHANGELOG.md

```diff
@@ -33,10 +36,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Removed
 - Enable backward compatibility tests on Mac ([#18983](https://github.com/opensearch-project/OpenSearch/pull/18983))
 
-### Fixed
```

Comment:
This section was duplicated so I removed the duplicated section and moved the entries above.